test/salt/states/setup-certs-to-remove.sls

```diff
@@ -13,6 +14,8 @@ add-test-cert:
         -----BEGIN CERTIFICATE-----
         3MOCKED CERT AND KEY
         -----END CERTIFICATE-----
+    - require:
+      - sls: {{ sls_package_install }}
```

Comment:
This isn't _strictly_ necessary but I thought it may be safer all round to ensure the packages are installed first, since there's no harm in that.